ui/StatusQ/include/StatusQ/modelutilsinternal.h

```diff
@@ -4,8 +4,8 @@
 #include <QObject>
 #include <QString>
 #include <QVariant>
+#include <QAbstractItemModel>
```

Comment:
Fixing an issue where the invokable signatures are not recognised in the qml tests